server/src/main/java/io/druid/indexing/overlord/autoscaling/ResourceManagementSchedulerConfig.java

```diff
@@ -28,6 +28,9 @@
 public class ResourceManagementSchedulerConfig
```

Comment:
we shouldn't have to move this class from indexing-service to server. There is no reason why all druid services would need to depend on this class. It is very much an indexing-service specific configuration.